xptifw/samples/syclur_collector/syclur_collector.cpp

```diff
@@ -32,15 +32,15 @@ XPTI_CALLBACK_API void xptiTraceInit(unsigned int major_version,
                                      const char *stream_name) {
   // The example basic collector under xptifw/samples/basic_collector takes in
   // streams from anyone as an example. In this collector, we will accept
-  // streams from just the SYCL  plugin interface (PI) layer.
+  // streams from just the SYCL Unified Runtime (UR) layer.
   printf("Stream Name: %s\n", stream_name);
   if (std::string("sycl.pi") == stream_name) {
```

Comment:
sycl.pi is not the case any more

Reply:
Thanks! I went ahead and done a full search and replace for `sycl.pi` to `ur.call` throughout the project.